jcache/commons/src/main/java/org/infinispan/jcache/AbstractJCacheNotifier.java

```diff
@@ -30,10 +31,10 @@
  * @author Galder Zamarreño
  * @since 5.3
  */
-public class JCacheNotifier<K, V> {
+public abstract class AbstractJCacheNotifier<K, V> {
```

Comment:
The subclasses look identical but createListenerAdapter() returns objects of different classes (same name but different package).